addons/project_mail_plugin/tests/test_controller.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+from unittest.mock import patch
+
+from odoo.addons.mail_plugin.tests.common import TestMailPluginControllerCommon
+
+
+class TestMailPluginProjectController(TestMailPluginControllerCommon):
+    def test_user_lang(self):
+        """Verify that we translate field in the user language."""
+        self.env["res.lang"]._activate_lang("fr_BE")
+        project = self.env["project.project"].create({"name": "Test Mail Plugin"})
+        project.with_context(lang="fr_BE").name = "[FR] Test Mail Plugin"
+        self.assertEqual(project.name, "Test Mail Plugin")
+        self.user_test.lang = "fr_BE"
```

Comment:
We could probably loop on lang = False, en_US, fr_BE, es_ES (not installed) and check result in all cases ? Just to cover: lang not set on user, current lang, installed lang, not installed lang ?

Reply:
for `es_ES` if it's not installed it will just crash when doing `self.user_test.lang = lang`, so I need to install it (but I can test the case where there's no translation for that lang)